src/diffusers/__init__.py

```diff
@@ -36,6 +36,15 @@
     "configuration_utils": ["ConfigMixin"],
     "guiders": [],
     "hooks": [],
+    "image_processor": [
```

Comment:
Is it ok to add `image_processor` and `video_processor` to `__init__.py`? Otherwise, my local preview of the docs won't build. No big deal either way though :)